src/server/system_services/bucket_server.js

```diff
@@ -985,7 +985,8 @@ async function list_buckets(req) {
     );
     return {
         buckets: _.map(buckets_by_name, function(bucket) {
-            return _.pick(bucket, 'name');
+            bucket.creation_date = bucket._id.getTimestamp().getTime();
+            return _.pick(bucket, ['name', 'creation_date']);
```

Comment:
this bucket object comes from system_store, and it is not a good idea not update it in-place.
instead just construct the answer object -
```suggestion
            return {
                name: bucket.name,
                creation_date: bucket._id.getTimestamp().getTime(),
            };
```

Reply:
Done, Thanks.